src/_playground/Routes.js

```diff
@@ -400,12 +400,12 @@ export class Routes extends Component {
                         </header>
                         <div className={sideBarClasses}>
                             <div className='frDocs-Search'>
-                                <InputGroup
-                                    glyph='search'
-                                    inputPlaceholder='Search'
-                                    inputProps={{ 'aria-label': 'Search' }}
-                                    inputValue={this.state.query}
-                                    onChange={this.onChangeHandler} />
+                                <InputGroup type='icon'>
+                                    <InputGroup.Addon>$</InputGroup.Addon>
```

Comment:
I think this was just an oversight - can it be changed back to the search sap icon?